### PR TITLE
Attempt at re-adding elasticache

### PR DESCRIPTION
**Pull request by cmc333333**

### CODE REVIEW

File: cloudformation.json [hunk 1]
```diff
@@ -688,6 +710,41 @@
...
+      "Type": "AWS::ElastiCache::SubnetGroup",
+      "Properties": {
+        "Description": "VPC subnets",
+        "SubnetIds": [
+          {"Ref": "PrivateSubnetA"},
+          {"Ref": "PrivateSubnetB"}
+        ],
+        "Tags" : [
+          {"Key" : "Application", "Value" : { "Ref" : "AWS::StackName"} },
+          {"Key" : "Name", "Value" : "Peace Corps DB Subnet Group" },
```
Comment (seanherron) [inline cloudformation.json]: Elasticcache Subnet Group

File: cloudformation.json [hunk 2]
```diff
@@ -688,6 +710,41 @@
...
+        "Engine": "memcached",
+        "CacheNodeType": "cache.t2.micro",
+        "CacheSubnetGroupName": { "Ref": "ElasticacheSubnetGroup" },
+        "NumCacheNodes": "2",
+        "VPCSecurityGroups": [
+          { "Ref" : "ElasticacheSecurityGroup" }
+        ],
+        "Tags" : [
+          {"Key" : "Application", "Value" : { "Ref" : "AWS::StackName"} },
+          {"Key" : "Name", "Value" : "Peace Corps Cache" },
```
Comment (seanherron) [inline cloudformation.json]: Elasticache

**Comment by seanherron** [comment]
How do we feel about a t2.micro cache? I'm not sure of the performance characteristics of memcached but t2.micros are pretty weak.

**Comment by cmc333333** [comment]
The entire site should be able to fit into a t2.micro a few times over. I think it'll be easy enough to change if we see performance degradation.

**Comment by seanherron** [comment]
@cmc333333 fair. Let's just change two of the nitpicky name entries and we can merge + deploy this.

**Comment by cmc333333** [comment]
Tested on staging - this appears to be working

**Comment by seanherron** [comment]
:+1:

**Pull request merged**